src/Raven.Server/Documents/DatabaseRequestHandler.cs

```diff
@@ -157,5 +165,68 @@ protected void AddPagingPerformanceHint(PagingOperationType operation, string ac
             if(ShouldAddPagingPerformanceHint(numberOfResults))
                 Database.NotificationCenter.Paging.Add(operation, action, details, numberOfResults, pageSize, duration, totalDocumentsSizeInBytes);
         }
+
+        protected DynamicJsonValue GetCustomConfigurationAuditJson(string name, BlittableJsonReaderObject configuration)
+        {
+            switch (name)
+            {
+                case "update-periodic-backup":
+                    return JsonDeserializationServer.BackupConfiguration(configuration).ToAuditJson();
+                case "etl-add":
+                    var etlType = EtlConfiguration<ConnectionString>.GetEtlType(configuration).ToString();
```

Comment:
lets compare the enum values, not strings, like here:
https://github.com/ravendb/ravendb/blob/1d56d2a9a84534fc3863e8a1da45b011eec882f1/src/Raven.Server/ServerWide/ServerStore.cs#L2211